### ISSUE TITLE
client_ident values are missing from OAuth test client tokens

### ISSUE THREAD

**Issue by phriscage** [CAAPIM/Docker-MAS#30]
## Issue/Request:
client_ident values are missing from OAuth test client tokens. When the *ADD_TEST_CLIENTS=true* environment variable is set, the *mas* container fails to start:

```
mas_1                | liquibase-mag-otk-db.sh: Created otk_db database on mysqldb
mas_1                | com.ca.liquibase-start.sh : Sourcing script /opt/docker/rc.d/liquibase/~liquibase-mag-otk-testdata-db.sh
mas_1                | Unexpected error running Liquibase: Error executing SQL INSERT INTO otk_db.oauth_token (otk_token_id, token, scope, status, client_name, expiration, resource_owner, client_key) VALUES ('access_token01', 'access_token01', 'scope01', 'ENABLED', 'client_name01', (unix_timestamp()+3600), 'testbuddy1', 'client_key01'): Field 'client_ident' doesn't have a default value
mas_1                |
mas_1                |
mas_1                | ERROR - Failed to create or update the otk_db database on mysqldb
dockermas_mas_1 exited with code 1
```

It looks like the client_ident attributes are missing from */db/liquibase/mag-otk-testdata.xml*. There are other liquibase xml files that are missing default values too.

Thanks,

Chris

**Comment by daudetCA** [comment]
Thanks @phriscage , I'll have a look at this.

**Comment by daudetCA** [comment]
@phriscage the liquibase test data scripts were missing some columns as you pointed out (`client_ident` for table `oauth_token` and `grace_expiration` for table `mag_msso_device`). I am working on the updates now and should have a new Docker image pushed to Docker Hub in the next day or so.

**Comment by kazvictor** [comment]
This was fixed with pr #32 The updated image can successfully create the test clients.
Note, for now this is only merged into develop. We will update master after reviewing/fixing issue #31

**Comment by kazvictor** [comment]
This was merged into master.
